Leakless causes panic on Windows

While trying to test out #55, I ran into an issue where it seems like Defender is finding a dependency and marking it as a virus.

```text
panic: fork/exec C:\Users\MIKEY~1\AppData\Local\Temp\leakless-0c3354cd58f0813bb5b34ddf3a7c16ed\leakless.exe: Operation did not complete successfully because the file contains a virus or potentially unwanted software.

goroutine 1 [running]:
github.com/go-rod/rod/lib/utils.glob..func2({0x1928c60?, 0xc0001126c0?})
        C:/Users/mlombardi/go/pkg/mod/github.com/go-rod/rod@v0.112.0/lib/utils/utils.go:60 +0x25
github.com/go-rod/rod/lib/utils.E(...)
        C:/Users/mlombardi/go/pkg/mod/github.com/go-rod/rod@v0.112.0/lib/utils/utils.go:66
github.com/go-rod/rod/lib/launcher.(*Launcher).MustLaunch(0xc00039c380?)
        C:/Users/mlombardi/go/pkg/mod/github.com/go-rod/rod@v0.112.0/lib/launcher/launcher.go:351 +0x67
main.New()
        C:/code/contrib/vhs/vhs.go:70 +0x27b
main.Evaluate({0xc0007ba6c0, 0x8b}, {0x1ea4ac0, 0xc000006018}, {0x0, 0x0, 0x0?})
        C:/code/contrib/vhs/evaluator.go:33 +0x1c5
main.glob..func1(0x2453ea0?, {0xc000497940, 0x1, 0x1?})
        C:/code/contrib/vhs/main.go:60 +0x16d
github.com/spf13/cobra.(*Command).execute(0x2453ea0, {0xc00005a3d0, 0x1, 0x1})
        C:/Users/mlombardi/go/pkg/mod/github.com/spf13/cobra@v1.6.0/command.go:916 +0x862
github.com/spf13/cobra.(*Command).ExecuteC(0x2453ea0)
        C:/Users/mlombardi/go/pkg/mod/github.com/spf13/cobra@v1.6.0/command.go:1040 +0x3b4
github.com/spf13/cobra.(*Command).Execute(...)
        C:/Users/mlombardi/go/pkg/mod/github.com/spf13/cobra@v1.6.0/command.go:968
main.main()
        C:/code/contrib/vhs/main.go:135 +0x25
```

Which seems like there's not a good fix (except maybe getting it signed/etc), but there _is_ a workaround:

> You can disable leakless, the browser process auto-kill won't work anymore, but you can add a defer on your main function to close the browser manually:
> 
> ```go
> u := launcher.New().Leakless(false).MustLaunch()
> 
> browser := rod.New().ControlURL(u).MustConnect()
> defer browser.MustClose()
> ```
> 
> _Originally posted by @ysmood in https://github.com/go-rod/rod/issues/739#issuecomment-1272420000_

--- a/evaluator.go
+++ b/evaluator.go
@@ -31,6 +31,7 @@ func Evaluate(tape string, out io.Writer, opts ...EvaluatorOption) error {
 	}
 
 	v := New()
+	defer func() { _ = v.close() }()
 
 	// Run Output and Set commands as they only modify options on the VHS instance.
 	var offset int

--- a/vhs.go
+++ b/vhs.go
@@ -24,6 +24,7 @@ type VHS struct {
 	mutex        *sync.Mutex
 	recording    bool
 	tty          *exec.Cmd
+	close        func() error
 }
 
 // Options is the set of options for the setup.
@@ -39,8 +40,10 @@ type Options struct {
 	Video         VideoOptions
 }
 
-const defaultFontSize = 22
-const typingSpeed = 50 * time.Millisecond
+const (
+	defaultFontSize = 22
+	typingSpeed     = 50 * time.Millisecond
+)
 
 // DefaultVHSOptions returns the default set of options to use for the setup function.
 func DefaultVHSOptions() Options {
@@ -64,7 +67,7 @@ func New() VHS {
 
 	opts := DefaultVHSOptions()
 	path, _ := launcher.LookPath()
-	u := launcher.New().Bin(path).MustLaunch()
+	u := launcher.New().Leakless(false).Bin(path).MustLaunch()
 	browser := rod.New().ControlURL(u).MustConnect().SlowMotion(opts.TypingSpeed)
 	page := browser.MustPage(fmt.Sprintf("http://localhost:%d", port))
 
@@ -77,6 +80,7 @@ func New() VHS {
 		tty:       tty,
 		recording: true,
 		mutex:     mu,
+		close:     browser.Close,
 	}
 }
 

